src/archiefvernietigingscomponent/demo/management/commands/create-demo-data.py

```diff
@@ -29,14 +30,21 @@ def handle(self, *args, **options):
         assert drc_service, "No service defined for the Documenten API"
         drc_client = drc_service.build_client()
 
-        # Create a catalogus
-        catalogus = ztc_client.create(resource="catalogus", data=CATALOGUS)
+        # Create a catalogus if one doesn't exist already
+        catalogi_list = ztc_client.list(resource="catalogus")
```

Comment:
Could you use the query parameters so that the filtering happens on the side of openzaak and either one or no catalogus is returned?